pom.xml

```diff
@@ -122,11 +122,11 @@
         <graalvm.version>22.3.0</graalvm.version>
         <gson.version>2.9.0</gson.version>
         <guice.version>4.0</guice.version>
-        <hadoop.version>3.3.0</hadoop.version>
+        <hadoop.version>3.3.1</hadoop.version>
```

Comment:
should this be 3.3.6

Reply:
I don't know how hard it will be. I see that we have changed the `hadoop.version` only on the root pom. There are 3 more pom files that are still using 3.1.1.